1_clusters/enable_mci.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -xe
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+if [ "$#" -lt 2 ]; then
+    >&2 echo "Not all expected arguments set."
+    exit 1
+fi
+
+PROJECT_ID=$1
+CLUSTER_NAMES=$2
+
+# IFS if we want to handle comma delimited list of clusters
+IFS="," 
+for CLUSTER_NAME in ${CLUSTER_NAMES}; do
+  gcloud alpha container hub ingress enable \
```

Comment:
I'm wondering if this capability should be included in the hub module?

Reply:
for MCI, this is specific to only the configuration cluster, it wouldn't apply to both clusters

Reply:
Interesting. It still seems like something we'd eventually want to add to the module, but I'm fine as is.